### PR TITLE
Fixed Gitcoinbot Documentation Content

### PR DESCRIPTION
**Pull request by molecula451**
##### Description

Our Section for GitcoinBot in the Documentation has some broken gif links which are intended to show our Bot in Action. We made some changes and update images so our users can actually see our examples.

**Actual Behaviour:**

issue

- Usage Example Gif images added

added

##### Testing

Chrome
Brave Browser

### CODE REVIEW

**Comment by molecula451** [comment]
@octavioamu @thelostone-mc Take a look..

File: docs/GITCOINBOT.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<a href="https://github.com/apps/gitcoinbot">
+    <img src="https://github.com/gitcoinco/web/raw/master/app/gitcoinbot/install.jpg">
+</a>
+
+## Usage instructions
+
+[Gitcoinbot](https://github.com/Gitcoinbot) is a bot that allows you to interact with gitcoin via the github comments, like follows:
+
+### Help Command
+![Gitcoinbot help](https://i.postimg.cc/tC2XHSFf/ezgif-1-defad716e014.gif)
```
Comment (danlipert) [inline docs/GITCOINBOT.md]: Can you include these gifs in the repo itself? Thanks!

Comment (molecula451) [reply docs/GITCOINBOT.md]: > Can you include these gifs in the repo itself? Thanks!

Sure thing my buddy. I was thinking something like this. We should not trust other hosting servers for our files/images

Comment (danlipert) [reply docs/GITCOINBOT.md]: @molecula451 Theres another one 2 lines down 🙇  thanks!

Comment (molecula451) [reply docs/GITCOINBOT.md]: @danlipert Uploaded and updated my friend. I have pointed out directly to the gitcoin/web repo. So at it soons it is merge it should detect the images that are in the original repo. Take a look

**Comment by molecula451** [comment]
@gitcoinbot submit work

**Comment by molecula451** [comment]
- Added Post Installation short Configuration Bot Handling, .gif

**Comment by molecula451** [comment]
@danlipert Who's mergin ?

**Comment by molecula451** [comment]
@danlipert Take a look mate

**Comment by molecula451** [comment]
@octavioamu Have take a look

**Comment by molecula451** [comment]
@thelostone-mc Review my friend

**Comment by molecula451** [comment]
@danlipert  Updated only content, leave the rest to mkdocs

File: app/gitcoinbot/README.md [hunk 2]
```diff
@@ -27,6 +27,12 @@ To get it running on your repo, you can install it [here](https://github.com/app
 
 Congrats on installing @gitcoinbot.  This bot facilitates gitcoin bounties via github commands.
 
+After Installation you can go ahead <a href="https://github.com/apps/gitcoinbot">here</a> again.
+
+![Gitcoinbot configure](https://github.com/gitcoinco/web/raw/master/docs/imgs/gitcoinbotconfigure.gif)
+
+From there you may select which repositorie you want the GitcoinBot to be working
```
Comment (danlipert) [inline app/gitcoinbot/README.md]: some grammar and spelling issues here: 

"After installation you can go here again"

"From there you may select which repository you want the Gitcoinbot to interact with"

Comment (danlipert) [reply app/gitcoinbot/README.md]: @molecula451 I think you missed the first bit of my wording change with "After installation you can go here again" - fix that and we're good to go!

Comment (molecula451) [reply app/gitcoinbot/README.md]: @danlipert I have fixed please see commits

Comment (molecula451) [reply app/gitcoinbot/README.md]: We are now instead using the Congrats on installing Gitcoinbot sentence and then...

"You can go ahead here again" where 'here' contains a direct link to the bot configuration panel

File: app/gitcoinbot/README.md [hunk 3]
```diff
@@ -27,6 +27,12 @@ To get it running on your repo, you can install it [here](https://github.com/app
 
 Congrats on installing @gitcoinbot.  This bot facilitates gitcoin bounties via github commands.
 
+You can go ahead <a href="https://github.com/apps/gitcoinbot">here</a> again.
```
Comment (molecula451) [inline app/gitcoinbot/README.md]: @danlipert Please see

File: app/gitcoinbot/README.md [hunk 4]
```diff
@@ -25,7 +25,13 @@ To get it running on your repo, you can install it [here](https://github.com/app
 
 ## Post Installation.
 
-Congrats on installing @gitcoinbot.  This bot facilitates gitcoin bounties via github commands.
+Congratulations on installing @gitcoinbot.  This bot facilitates gitcoin bounties via github commands.
+
+You can go <a href="https://github.com/apps/gitcoinbot">here</a>.
```
Comment (molecula451) [inline app/gitcoinbot/README.md]: @danlipert Please see..

Comment (molecula451) [reply app/gitcoinbot/README.md]: Still seeing something is left?

Comment (molecula451) [reply app/gitcoinbot/README.md]: @danlipert Let's get this done before next year....

**Comment by molecula451** [comment]
@danlipert Can we end up this? can you please double-check all the most recent fixes?

**Comment by molecula451** [comment]
@danlipert Up to you...

**Comment by molecula451** [comment]
@danlipert I will be happy if you close this asap so I can clean up my repo

**Pull request merged**